SAMtemplates/state_machines/GetMyPrescriptionsStateMachine.asl.json

```diff
@@ -9,17 +9,107 @@
         "Payload.$": "$",
         "FunctionName": "${GetMyPrescriptionsFunctionArn}"
       },
+      "Next": "Get My Prescriptions Result",
+      "Catch": [
+        {
+          "ErrorEquals": [
+            "States.ALL"
+          ],
+          "Next": "Catch All Error"
+        }
+      ]
+    },
+    "Get My Prescriptions Result": {
+      "Type": "Choice",
+      "Choices": [
+        {
+          "Not": {
+            "Variable": "$.Payload.statusCode",
+            "NumericEquals": 200
+          },
+          "Next": "Failed Get My Prescriptions"
+        }
+      ],
+      "Default": "Parse Get My Prescriptions Body"
+    },
+    "Parse Get My Prescriptions Body": {
+      "Type": "Pass",
+      "Next": "Get Toggle Get Status Updates Parameter",
+      "InputPath": "$.Payload.body",
+      "Parameters": {
+        "body.$": "States.StringToJson($)"
+      },
+      "OutputPath": "$.body"
+    },
+    "Get Toggle Get Status Updates Parameter": {
+      "Type": "Task",
+      "Next": "Evaluate Toggle Get Status Updates Parameter",
+      "Parameters": {
+        "Name": "${ToggleGetStatusUpdatesParameterName}"
+      },
+      "Resource": "arn:aws:states:::aws-sdk:ssm:getParameter",
+      "ResultPath": "$.ToggleGetStatusUpdatesParameter"
```

Comment:
should have a catch statement in this task

Reply:
Removing this task shortly.